hosts/common.nix

```diff
@@ -110,6 +110,15 @@ in
   # Shell
   programs.bash.completion.enable = true;
 
+  systemd.services.trim_profiles = {
```

Comment:
Suggested change (line 113):
```
  systemd.services.trim-profiles = {
```
That is, rename the service to 'trim-profiles' to follow the naming convention used with other systemd services.